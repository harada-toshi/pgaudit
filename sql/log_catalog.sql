\! cp sql/log_catalog/log_catalog_default.conf /tmp/config.conf
\! echo "pgaudit.config_file = '/tmp/config.conf'" >> ./tmp_check/data/postgresql.conf
\! rm ./tmp_check/tmp_check.log

\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/log_catalog/log_catalog.sql
\! rm /tmp/config.conf

\! cp sql/log_catalog/log_catalog_on.conf /tmp/config.conf
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/log_catalog/log_catalog.sql
\! rm /tmp/config.conf

\! cp sql/log_catalog/log_catalog_off.conf /tmp/config.conf
\! pg_ctl restart -w -D ./tmp_check/data -l ./tmp_check/tmp_check.log
\! psql contrib_regression -q -a < sql/log_catalog/log_catalog.sql
\! rm /tmp/config.conf

\! grep "AUDIT" ./tmp_check/tmp_check.log > results/log_catalog.out

