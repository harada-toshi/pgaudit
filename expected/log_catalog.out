		format='AUDIT: %object_id,%command_text'
LOG:  AUDIT: ,SELECT 1;
LOG:  AUDIT: pg_catalog.pg_am,SELECT * FROM pg_am;
		format='AUDIT: %object_id,%command_text'
LOG:  AUDIT: ,SELECT 1;
LOG:  AUDIT: pg_catalog.pg_am,SELECT * FROM pg_am;
		format='AUDIT: %object_id,%command_text'
LOG:  AUDIT: ,SELECT 1;
